addons/l10n_sa_edi/models/account_edi_xml_ubl_21_zatca.py

```diff
@@ -115,6 +115,11 @@ def _get_invoice_payment_means_vals_list(self, invoice):
         res[0]['adjustment_reason'] = invoice.ref
         return res
 
+    def _get_partner_contact_vals(self, partner):
```

Comment:
please mention in the commit that we added some backport

Reply:
done